### DISCUSSION TITLE
在neovim中输入中文时，使用退格键把之前输入的字符都给删除掉了。而不是只删除输入未确定的中文。

### DISCUSSION THREAD

**Discussion by xclidongbo** [rime/home#1261] [Q&A]
在neovim中输入中文时，使用退格键把之前输入的字符都给删除掉了。而不是只删除输入未确定的中文。

**Comment by qujihan** [comment]
是不是使用的alacritty？我原来使用alacritty的时候也是这样， 这是alacritty的原因

**Reply by xclidongbo** [reply]
我还以为是输入法的问题呢。。
